### DISCUSSION TITLE
running jobs via HTTP asynchronously

### DISCUSSION THREAD

**Discussion by agurnani-mwh** [jhuckaby/Cronicle#713] [Q&A]
HTTP plugin allows me to submit an HTTP request. I want to know if I can receive a job response asynchronously. For long running jobs, the http endpoint will spawn a thread and return a response of 200 before the job is done and then when the job is done, I want to respond back to Cronicle so it can do all the other cool things (like update job status, notify users of success/fail, kick of the next job in chain etc.)

so far I have not found any way to do that. Any help will be greatly appreciated.

Regards
Anil

**Comment by jhuckaby** [comment]
You need to write a Plugin to handle a custom case like that: https://github.com/jhuckaby/Cronicle/blob/master/docs/Plugins.md
